### PR TITLE
ELPP-3598 Healthcheck real world feedback

### PR DESCRIPTION
**Pull request by giorgiosironi**
New Relic probes from Brazil and Japan fail completely due to the local Fastly servers not reaching the origin within the healthcheck timeouts.

### CODE REVIEW

**Comment by giorgiosironi** [comment]
Much better now from all locations:

File: projects/elife.yaml [hunk 1]
```diff
@@ -370,9 +370,11 @@ api-gateway:
                 gcslogging:
                     bucket: end2end-elife-fastly
                     path: api-gateway--test/
-                    period: 1800
+                    period: 600
                 healthcheck:
                     path: /ping-fastly
+                    check-interval: 30000
+                    timeout: 10000
```
Comment (thewilkybarkid) [inline projects/elife.yaml]: 10 seconds seems a bit long, looks like it was 0.5 before?

Comment (giorgiosironi) [reply projects/elife.yaml]: can be tuned, this is a large precautionary value that makes the POPs from around the world happy

**Comment by giorgiosironi** [comment]
Just applied, https://synthetics.newrelic.com/accounts/1451451/monitors/375649ae-c489-4ea6-8ddf-bbe7f30f9a2b will tell us the result.

**Comment by giorgiosironi** [comment]
New Relic shows no failures in the last 3 hours, so we can merge this.

**Comment by giorgiosironi** [comment]
For the record, the slowest result is from Mumbai with 1.67 seconds for the NewRelic->FastlyPOP->AWS path.

**Pull request merged**